eventGenerator/main.go

```diff
@@ -24,8 +24,8 @@ func generate(seconds int, functions ...func()) {
 
 func main() {
 	gotenv.Load()
-	generate(5, GenerateWaterQualityEvent, GenerateWindEvent, GenerateCarbonEvent)
-	generate(60, GenerateWaterConsumptionEvent)
+	generate(5, GenerateWaterQualityEvent, GenerateWindEvent, GenerateCarbonEvent, GenerateTemperatureEvent)
+	generate(60, GenerateWaterMeterEvent)
```

Comment:
Does this build? `GenerateWaterMeterEvent` doesn't exist anymore, I suppose, it's renamed